extensions/wikia/CoppaTool/legacy/IpAnonymizer.php

```diff
@@ -7,6 +7,7 @@ public function anonymizeIp( string $ip ): void {
 		$dbw = wfGetDB( DB_MASTER );
 		$this->updateTable( $dbw, 'revision', 'rev_user_text', $ip );
 		$this->updateTable( $dbw, 'recentchanges', 'rc_user_text', $ip );
+		$this->updateTable( $dbw, 'recentchanges', 'rc_ip_bin', $ip );
```

Comment:
On App `rc_ip_bin` is encoded to binary format like `4 B00000000  95 06 1C A4 ...¤`. We need to convert it
```suggestion
		$this->updateTable( $dbw, 'recentchanges', 'rc_ip_bin', inet_pton( $ip ) );
```

Reply:
Reference example: [app/includes/RecentChange.php](https://github.com/Wikia/app/blob/33430545d9e55461becc89b73fdbf81ea25ef240/includes/RecentChange.php#L432)